tools.go

```diff
@@ -0,0 +1,7 @@
+// +build tools
+
+package tools
+
+import (
+	_ "sigs.k8s.io/controller-tools/pkg/crd/generator"
+)
```

Comment:
The reason of this file is to generate the `crd` once we update the `types` scheme?

Reply:
This file is part of the scaffold generated by the operator-sdk when go-mod is enabled. It is used for tool dependencies when running the `operator-sdk generate` command with operator-sdk v0.9.0. The details are discussed in the pull requests: [go mod support for new operators 1](https://github.com/operator-framework/operator-sdk/pull/1001#issuecomment-464826207), [go mod support for new operators 2](https://github.com/operator-framework/operator-sdk/pull/1001#discussion_r251139696) and [tool dependencies](https://github.com/golang/go/issues/25922#issuecomment-402918061).